grammar/SignalFlowV2Parser.g4

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright (C) 2016 SignalFx, Inc.
+ */
+parser grammar SignalFlowV2Parser;
+
+options { tokenVocab = SignalFlowV2Lexer; }
+tokens { INDENT, DEDENT }
+
+program
+  : ( NEWLINE | statement )* EOF
+  ;
+
+functionDefinition
+ : DEF ID parameters ':' suite
+ ;
+
+parameters
+ : OPEN_PAREN varArgsList? CLOSE_PAREN
+ ;
+
+varArgsList
+ : varArgsListParamDef ( ',' varArgsListParamDef )*
```

Comment:
What's the policy on what tokens get added to the lexer file, and what ones are inlined in here (like `','` here, and `'='` and `';'` below)?